json_serializable/tool/shared.dart

```diff
@@ -8,7 +8,8 @@ import 'package:build/build.dart';
 import 'package:dart_style/dart_style.dart';
 import 'package:yaml/yaml.dart';
 
-final formatter = DartFormatter();
+final formatter =
+    DartFormatter(languageVersion: DartFormatter.latestLanguageVersion);
```

Comment:
Just realized this is pointless, since `latestLanguageVersion` is the default. 🤷 🤦

Reply:
You also really don't want to do it this way. I am prepping a change to source_gen which is going to be breaking, and will give you a language version to use in the format function.